Src/BootCamp.Chapter/FileCleaner.cs

```diff
@@ -0,0 +1,218 @@
+﻿using System;
+using System.IO;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    public static class FileCleaner
+    {
+        private const char DollarSign = '£';
+
+        /// <summary>
+        /// Cleans up dirtyFileName 
+        /// </summary>
+        /// <param name="dirtyFile">Dirty file with "_" placed in random places.</param>
+        /// <param name="cleanedFile">Cleaned up file without any "_".</param>
+        public static void Clean(string dirtyFile, string cleanedFile)
+        {
+            if (string.IsNullOrEmpty(dirtyFile))
+            {
+                throw new ArgumentException();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use a constructor overloads that allows a more meaningful exception message to be provided.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5116146)